server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/UnsetDefaultAction.java

```diff
@@ -49,9 +50,7 @@ public void define(WebService.NewController controller) {
   }
 
   @Override
-  public void handle(Request request, Response response) {
-    qualityGates.setDefault(null);
-    response.noContent();
+  public void handle(Request request, Response response) throws Exception {
```

Comment:
It order to avoid this empty method you have 2 choices : 
- Move RemovedWebServiceHandler.INSTANCE.handle(request, response) here
- Delete this class and add the declaration of this action in QualityGatesWs (see [example](https://github.com/SonarSource/sonarqube/blob/master/server/sonar-server/src/main/java/org/sonar/server/component/ws/ResourcesWs.java#L36))